recipes/elf/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "ed00f76b0880aed3943de4c66cc70a3946827f3662f85f8b2c5920bab23000ba" %}
+{% set name = "elf" %}
+
+package:
+    name: {{ name|lower }}
```

Comment:
`python-elf` here? `elf` means `Executable and Linkable Format` usually in packaging sense.